packages/bpk-component-page-indicator/src/BpkPageIndicator.js

```diff
@@ -0,0 +1,146 @@
+  overImage: 'overImage',
+};
+
+export type Props = {
+  className: ?string,
+  showNav: ?boolean,
+  currentIndex: number,
+  totalIndicators: number,
+  onClick: ?() => void,
+  indicatorLabel: string,
+  prevNavLabel: string,
+  nextNavLabel: string,
+  style: ?$Values<typeof INDICATOR_STYLE_TYPE>,
+};
+
+const BpkPageIndicator = ({
+  className,
+  currentIndex,
+  indicatorLabel,
+  nextNavLabel,
+  onClick,
+  prevNavLabel,
+  showNav,
+  style,
+  totalIndicators,
+}: Props) => (
+  // $FlowFixMe[cannot-spread-inexact] - inexact rest. See 'decisions/flowfixme.md'.
+  <div className={className}>
+    <div
+      className={getClassName(
+        'bpk-page-indicator',
+        showNav && 'bpk-page-indicator__showNav',
+      )}
+    >
+      {showNav && (
+        <NavButton
+          currentIndex={currentIndex}
+          onClick={onClick}
+          disabled={currentIndex === 0}
+          prev
+          ariaLabel={prevNavLabel}
+        />
+      )}
+      <div className={getClassName('bpk-page-indicator__container')}>
+        <div
+          className={getClassName('bpk-page-indicator__indicators-container')}
+          style={
+            currentIndex > START_SCROLL_INDEX
+              ? {
+                  '--scroll-index':
+                    totalIndicators > DISPLAYED_TOTAL
+                      ? Math.min(
+                          currentIndex - START_SCROLL_INDEX,
+                          totalIndicators - DISPLAYED_TOTAL,
+                        )
+                      : 0,
+                }
+              : undefined
+          }
+        >
+          {[...Array(totalIndicators)].map((val, index) => (
+            <button
+              type="button"
+              onClick={(e) => {
+                onClick(e, index, DIRECTIONS.INDICATORS);
+              }}
+              className={getClassName(
+                'bpk-page-indicator__indicator',
+                `bpk-page-indicator__indicator--${style}`,
+                index === currentIndex &&
+                  `bpk-page-indicator__indicator--active-${style}`,
+              )}
+              aria-label={`${indicatorLabel} ${index + 1}`}
+              aria-pressed={currentIndex === index ? 'true' : 'false'}
+              // eslint-disable-next-line react/no-array-index-key
+              key={`indicator-${index}`}
+            />
+          ))}
+        </div>
+      </div>
+      {showNav && (
+        <NavButton
+          currentIndex={currentIndex}
+          onClick={onClick}
+          disabled={currentIndex === totalIndicators - 1}
+          ariaLabel={nextNavLabel}
+        />
+      )}
+    </div>
+  </div>
+);
+
+BpkPageIndicator.propTypes = {
+  indicatorLabel: PropTypes.string.isRequired,
+  prevNavLabel: PropTypes.string.isRequired,
+  nextNavLabel: PropTypes.string.isRequired,
+  currentIndex: PropTypes.number.isRequired,
+  totalIndicators: PropTypes.number.isRequired,
+  style: PropTypes.oneOfType(Object.values(INDICATOR_STYLE_TYPE)),
```

Comment:
I think it would be more common to use `Object.keys`

Reply:
Have updated `$Values `-> `$Keys`, `Object.values` ->` Object.keys`. Thank you for your patient review!